src/azul/__init__.py

```diff
@@ -1772,45 +1772,66 @@ def docker_image_gists_path(self) -> Path:
 
     blocked_user_agents_custom_regex_term = 'blocked_user_agents_custom'
 
-    waf_rate_rule_name = 'RateRule'
+    waf_rate_rule_name = 'rate_limit'
 
-    waf_rate_alarm_rule_name = 'RateAlarmRule'
+    waf_rate_alarm_rule_name = 'rate_limit_alarm'
 
-    waf_rate_rule_period = 300  # seconds; this value is fixed by AWS
+    @frozen(kw_only=True)
+    class RateLimit:
+        #: Name of the WAF rule
+        name: str
 
-    waf_rate_rule_retry_after = 30  # seconds
+        #: Number of requests per evaluation period
+        value: int
 
-    waf_rate_rule_limit = 1000
+        #: Evaluation window in seconds. Note, the rate is checked every 10
+        #: seconds. This property specifies how far back from the current time
+        #: to consider when calculating the rate.
+        period: int
 
-    @frozen(kw_only=True)
-    class FileDownloadLimit:
-        rate_limit: int
-        evaluation_window: int
-        assumed_request_concurrency: float
+        #: Value of the Retry-After header, in seconds, sent back to requests
+        # blocked by the rate rule
+        retry_after: int
 
-        @classmethod
-        def parse(cls, s: str) -> Self:
-            rate, s = s.split('/')
-            window, concurrency = s.split('@')
-            return cls(rate_limit=int(rate),
-                       evaluation_window=int(window),
-                       assumed_request_concurrency=float(concurrency))
+        def __attrs_post_init__(self):
+            # Allowed range of the rate limit mandated by AWS
+            assert 10 <= self.value <= 2_000_000_000, R(
+                'Rate limit out of range', self)
+            # Valid values for the period mandated by AWS
+            assert self.period in [60, 120, 300, 600], R(
+                'Evaluation window specified is not valid', self)
```

Comment:
```suggestion
                'Invalid period', self)
```

The rest is just noise.